src/box/memtx_tree.cc

```diff
@@ -439,6 +469,9 @@ tree_iterator_prev_equal_base(struct iterator *iterator, struct tuple **ret)
 	}
 	struct index *idx = iterator->index;
 	struct space *space = space_by_id(iterator->space_id);
+
+/***********MVCC TRANSACTION MANAGER STORY GARBAGE COLLECTION BOUND************/
```

Comment:
I'd move this comment right after brace closing if else block. Also I'd change these lines to "start of the bound" and "end of the bound" to improve their readability:)

Reply:
> I'd move this comment right after brace closing if else block
Could you elaborate, please? I don't understand the point.

> Also I'd change these lines to "start of the bound" and "end of the bound" to improve their readability
Fixed.